src/hypofuzz/database.py

```diff
@@ -404,6 +401,10 @@ def save_observation(
             obs_buffer.extend(
                 sorted(
                     list(self.fetch_observations(key)),
+                    # TODO run_start is the same for all reports from a worker,
+                    # since we use one StateForActualGivenExecution per FuzzProcess.
+                    # should we add a new `timestamp` hypofuzz-specific metadata?
+                    # (see also DataProvider code which relies on run_start).
```

Comment:
Somewhere I proposed overwriting the `run_start` with the `timestamp` from Hypofuzz; the former isn't really well-defined in the fuzzing setting.